lib/gtfs_realtime_viz.ex

```diff
@@ -94,6 +156,17 @@ defmodule GTFSRealtimeViz do
     end)
   end
 
+  @spec format_times([String.t] | nil) :: [String.t]
```

Comment:
I think we had talked about wanting the headsigns, too. If you don't want to make that part of this ticket, can you add a new asana task for it?

Reply:
oh, i wasnt sure where we landed on that - im actually not really sure how important headsigns really are, when we are looking at trains arriving at specific stations.  i was thinking we care mostly about if our predictions are right at a given stop, and seeing the time and a train arriving gets us there.  does having a headsign add something else that we need?

Reply:
Eh, it's more data and _might_ be useful. Without having thought about it too hard, I feel like it could help us make sure we're not mixing up trips somewhere? We definitely don't need it now, but I think it's worth at least discussing as a team, which is what the asana task would provide.

Reply:
ok, ill add the task.

Reply:
https://app.asana.com/0/373237768778637/509290385214342